pom.xml

```diff
@@ -300,7 +300,7 @@
                 <plugin>
                     <groupId>org.owasp</groupId>
                     <artifactId>dependency-check-maven</artifactId>
-                    <version>4.0.0</version>
+                    <version>5.0.0-M2</version>
```

Comment:
v5 is out

Reply:
I don't understand your comment: the point of this PR is precisely to update `dependency-check-maven` to version 5

Reply:
Never mind I misunderstood something. I meant that m2 was not up to date.